lib/process-services-cloud/src/lib/rich-text-editor/rich-text-editor.component.stories.ts

```diff
@@ -17,7 +17,7 @@
 
 import { RichTextEditorModule } from './rich-text-editor.module';
 import { Meta, moduleMetadata, Story } from '@storybook/angular';
-import { CoreStoryModule } from '../testing/core.story.module';
+import { CoreStoryModule } from '../../../../core/src/lib/testing/core.story.module';
```

Comment:
Can we import the ProcessServicesCloudStoryModule instead of the CoreStoryModule?